internal/resources/acceptance_tests/audit_log_export_acceptance_test.go

```diff
@@ -0,0 +1,222 @@
+	if err != nil {
+		t.Error(err)
+		t.FailNow()
+	}
+	auditLogSettingsResourceName := "acc_audit_log_settings_" + acctest.GenerateRandomResourceName()
+	auditLogExportResourceName := "acc_audit_log_export_" + acctest.GenerateRandomResourceName()
+	auditLogExportResourceReference := "couchbase-capella_audit_log_export." + auditLogExportResourceName
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.TestAccPreCheck(t) },
+		ProtoV6ProviderFactories: acctest.TestAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAuditLogExportSetup(acctest.Cfg, projectName, projectResourceReference, clusterResourceName, clusterResourceReference, cidr, auditLogSettingsResourceName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccAuditLogExportGetCluster(clusterResourceReference),
+				),
+			},
+			{
+				Config: testAccAuditLogExportConfig(acctest.Cfg, projectName, projectResourceReference, clusterResourceName, clusterResourceReference, cidr, auditLogExportResourceName, auditLogExportResourceReference),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr(auditLogExportResourceReference, "expiration", ""),
+					resource.TestCheckResourceAttr(auditLogExportResourceReference, "audit_log_download_url", ""),
+					resource.TestCheckResourceAttr(auditLogExportResourceReference, "status", ExportStatusNoAuditLogs),
+				),
+			},
+		},
+	})
+}
+
+// create cluster and enable audit logs
+func testAccAuditLogExportSetup(providerAndVariables, projectName, projectResourceReference, clusterResourceName, clusterResourceReference, cidr, auditLogSettingsResourceName string) string {
+	config := fmt.Sprintf(`
+%[1]s
+
+resource "couchbase-capella_project" "%[2]s" {
+    organization_id = var.organization_id
+	name            = "%[2]s"
+	description     = "terraform audit log export test cluster"
+}
+
+resource "couchbase-capella_cluster" "%[4]s" {
+  organization_id = var.organization_id
+  project_id      = %[3]s.id
+  name            = "terraform audit log export test cluster"
+  description     = "terraform audit log export test cluster"
+  configuration_type = "multiNode"
+  cloud_provider = {
+    type   = "aws"
+    region = "us-east-1"
+    cidr   = "%[6]s"
+  }
+  service_groups = [
+    {
+      node = {
+        compute = {
+          cpu = 4
+          ram = 16
+        }
+        disk = {
+          storage = 50
+          type    = "gp3"
+          iops    = 3000
+        }
+      }
+      num_of_nodes = 3
+      services     = ["data"]
+    }
+  ]
+  availability = {
+    "type" : "multi"
+  }
+  support = {
+    plan     = "enterprise"
+    timezone = "PT"
+  }
+}
+
+resource "couchbase-capella_audit_log_settings" "%[7]s" {
+  organization_id   = var.organization_id
+  project_id        = %[3]s.id
+  cluster_id        = %[5]s.id
+  audit_enabled     = true
+  enabled_event_ids = [8243, 8257, 8265]
+  disabled_users    = []
+}
```

Comment:
This resource needs to be added to "testAccAuditLogExportConfig"; otherwise, Terraform detects it as a delete operation and tries to call delete first for audit_log_settings.